queries/landuse.jinja2

```diff
@@ -53,21 +80,8 @@ FROM
 WHERE
     mz_is_landuse = TRUE
     AND {{ bounds|bbox_filter('way') }}
-
-    {% if zoom == 9 %}
-    AND way_area::bigint > 409600
-    {% elif zoom == 10 %}
-    AND way_area::bigint > 102400
-    {% elif zoom == 11 %}
-    AND way_area::bigint > 25600
-    {% elif zoom == 12 %}
-    AND way_area::bigint > 6400
-    {% elif zoom == 13 %}
-    AND way_area::bigint > 1600
-    {% elif zoom == 14 %}
-    AND way_area::bigint > 400
-    {% elif zoom == 15 %}
-    AND way_area::bigint > 100
+    {% if zoom <= 15 %}
+    AND (17.256-ln(way_area)/ln(4)) <= {{ zoom }}
```

Comment:
Yup, good point! I've added a function for that in b8e748d which hopefully makes it more readable while still being succinct.